src/components/TextField/TextField.jsx

```diff
@@ -1,13 +1,30 @@
 import React from 'react';
-import { Input } from './style';
+import PropTypes from 'prop-types';
+import { Input, P } from './style';
 
-function TextField(prop) {
-  const { defaultValue, disabled, redBorder } = prop;
+const TextField = (props) => {
+  const {
+    value, disabled, onChange, error, onBlur,
+  } = props;
   return (
     <>
-      <Input type="text" defaultValue={defaultValue} disabled={disabled} redBorder={redBorder} />
+      <Input type="text" defaultValue={value} disabled={disabled} onChange={onChange} error={error} onBlur={onBlur} />
```

Comment:
accept two props defaultValue and value
And use the values to corresponding props for Input tag 
(defaultValue={defaultValue} value={value} )